bin/predict_cellassign.py

```diff
@@ -91,19 +91,34 @@
 # note that the gene names must be the rownames of the reference matrix
 # first get a list of shared genes
 shared_genes = list(set(ref_matrix.index) & set(annotated_adata.var_names))
+
+# check that shared_genes actually has some genes
+if len(shared_genes) == 0:
```

Comment:
V. minor, but we could do this a bit more pythonically. 
```suggestion
if not shared_genes:
```